mlir/include/mlir/Dialect/OpenMP/OpenMPOps.td

```diff
@@ -920,6 +663,9 @@ def TaskloopOp : OpenMP_Op<"taskloop", [AttrSizedOperandSegments,
     specifies how to combine the values from each iteration into the final
     value, which is available in the accumulator after the loop completes.
 
+    The optional `byref` attribute controls whether reduction arguments are
+    passed by reference or by value.
+
```

Comment:
Here too. I guess we need separate work to standardize how reductions work over different ops?